src/main/java/com/epam/test_generator/controllers/testresult/response/CaseResultDTO.java

```diff
@@ -1,6 +1,5 @@
-package com.epam.test_generator.dto;
+package com.epam.test_generator.controllers.testresult.response;
```

Comment:
I think we should make it `com.epam.test_generator.controllers.test.result.response`